pkg/counter/cache_test.go

```diff
@@ -141,10 +141,12 @@ func TestInitializeCounterFromToolchainCluster(t *testing.T) {
 		HasUserAccountCount("member-2", 3)
 }
 
-func TestInitializeCounterFromToolchainClusterWithNegativeNumbersInCache(t *testing.T) {
+func TestInitializeCounterFromToolchainClusterWithoutReset(t *testing.T) {
 	// given
 	counter.DecrementUserAccountCount(logger, "member-1")
 	counter.DecrementMasterUserRecordCount(logger)
+	counter.UpdateUsersPerActivationCounters(1) // a user signup once
+	counter.UpdateUsersPerActivationCounters(2) // a user signup twice (hence counter for "1" will be decreased)
```

Comment:
This is a test case covering the initialization, but there is no test that would be focused purely on this functionality - could you please add some. Something similar to what is already there for the other counters

Reply:
done in e41cacb